Snake/SnakeTest/Maze_Test.cs

```diff
@@ -54,5 +54,24 @@ public void Istrue_return_a_maze_with_border_from_CreateMaze_function()
         }
 
 
+        [TestMethod]
+
+
+        public void IsMistake_Generate_Obstacles_In_Maze()
```

Comment:
test name does not seem to match what you are asserting. the name needs to be more specific